crates/swc_ecma_minifier/tests/fixture/issues/moment/1/output.js

```diff
@@ -1274,7 +1271,7 @@
         }
         return asFloat ? output : absFloor(output);
     }, proto.endOf = function(units) {
-        var time, startOfDate;
+        var time, startOfDate, divisor, divisor1, divisor2;
```

Comment:
I think we should consider cost of this decls

Reply:
Maybe not. Terser could optimize this away.

Reply:
... But only when param is a literal. So ultimately this is decided on the caller side, while we calcuate inline on the callee side.

Reply:
So we need a better analyzer?
I think we should count them as 1 (+1 for comma) for now.

Reply:
Wait, I got a idea...

Reply:
This is still unresolved because of #5367...